java_io_block/optional_tasks/data/javaFileForTask9.java

```diff
@@ -0,0 +1,35 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Task1 {
```

Comment:
please leave the comment that this is not actual java code file and it used only for testing